tests/src/unit-windows_h.cpp

```diff
@@ -7,6 +7,8 @@
 // SPDX-License-Identifier: MIT
 
 #include "doctest_compatibility.h"
+#undef WIN32_LEAN_AND_MEAN
+#undef NOMINMAX
```

Comment:
doctest defines these and doesn't restore them, defeating one of the purposes of the unit test.
(I.e., finding unparanthesized uses of `std::numeric_limits<T>::min()`, etc. in `json.hpp`.)

Reply:
I've submitted a PR to doctest.